.github/workflows/ci.yml

```diff
@@ -60,6 +60,9 @@ jobs:
           - test:one ember-lts-3.4
           - test:one ember-lts-3.8
           - test:one ember-lts-3.12
```

Comment:
Do we still want to keep these? I think the idea was to test the last 3 lts releases.

Reply:
I think I'd test all LTS releases that we support (until that leads to problems when we're using too many GitHub Actions resources or so).